airflow/operators/slack_operator.py

```diff
@@ -16,10 +16,12 @@
 from airflow.models import BaseOperator
 from airflow.utils.decorators import apply_defaults
 from airflow.exceptions import AirflowException
+from airflow.hooks.slack_webhook_hook import SlackWebHookHook
 import json
 import logging
 
 
+
```

Comment:
This is redundant empty line according to pep8 (need to have 2 empty line before class declaration).